microsoft/testsuites/kselftest/kselftest.py

```diff
@@ -92,33 +92,50 @@ def _check_exists(self) -> bool:
         )
 
     def __init__(
-        self, node: Node, kselftest_file_path: str, *args: Any, **kwargs: Any
+        self,
+        node: Node,
+        kselftest_working_path: str,
```

Comment:
`working_path` and `file_path`. It's in the kselftest class already, it doesn't need to have the name again.